js/whosehouse.js

```diff
@@ -64,11 +65,13 @@ function displayWhoseHouseWinner() {
   $('#wheel').bind(transitionEndEvents,
       (function(winnerHouse) {
         return function() {
+          $('.wheel-check').attr('disabled', false);
```

Comment:
actually you want to just remove the attribute altogether: `.removeAttr('disabled')` and to add it back `.attr('disabled', 'disabled')` or something like that.